modular_skyrat/master_files/code/game/objects/items/devices/radio/headset.dm

```diff
@@ -3,3 +3,38 @@
 	desc = "The headset of the king (or queen) of paperwork."
 	icon_state = "com_headset"
 	keyslot = new /obj/item/encryptionkey/heads/qm
+
+/obj/item/radio/headset/heads/rd/alt
+	name = "\proper the research director's bowman headset"
+	desc = "Headset of the fellow who keeps society marching towards technological singularity. Protects ears from flashbangs."
+	icon_state = "com_headset_alt"
+
+/obj/item/radio/headset/heads/rd/alt/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
+
+/obj/item/radio/headset/heads/ce/alt
+	name = "\proper the chief engineer's bowman headset"
+	desc = "The headset of the guy in charge of keeping the station powered and undamaged. Protects ears from flashbangs."
+	icon_state = "com_headset_alt"
+
+/obj/item/radio/headset/heads/ce/alt/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
+
+/obj/item/radio/headset/heads/cmo/alt
+	name = "\proper the chief medical officer's bowman headset"
+	desc = "The headset of the highly trained medical chief. Protects ears from flashbangs."
+	icon_state = "com_headset_alt"
+
+/obj/item/radio/headset/heads/cmo/alt/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
+/obj/item/radio/headset/heads/qm/alt
+	name = "\proper the quartermaster's bowman headset"
+	desc = "The headset of the king (or queen) of paperwork. Protects ears from flashbangs."
+	icon_state = "com_headset_alt"
+
+/obj/item/radio/headset/heads/qm/alt/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
```

Comment:
First of all, those should probably be called `/bowman` rather than `/alt` because that doesn't mean anything.

Secondly, I too think this is some unnecessary powercreep and I don't really see a need for this in the game.

Reply:
I did alt because that's what the other bowmans were called so that was... the thing to do.